src/react/src/features/auth/Login.tsx

```diff
@@ -21,49 +21,71 @@ export function Login() {
     if (isLoggedIn) {
         return <Redirect to="/collection" />;
     }
+
     console.log(message, status === 'failed');
 
     return (
-        <Box component="form" autoComplete="off" onSubmit={handleSubmit}
-            sx={{
-                display: 'flex',
-                flexDirection: 'column',
-                alignItems: 'center',
-                mt: 4
-            }}>
-            <Typography component="h1" variant="h5">
-                Sign in
-            </Typography>
-            <TextField
-                id="email"
-                label="Email"
-                placeholder="Please enter your email"
-                value={email}
-                onChange={(e) => handleInput(e, setEmail)}
-                margin="normal"
-                required
-                type="email"
-                error={status === 'failed'}
-            />
-            <TextField
-                id="password"
-                label="Password"
-                placeholder="Please enter your password"
-                value={password}
-                type="password"
-                onChange={(e) => handleInput(e, setPassword)}
-                margin="normal"
-                required
-                error={status === 'failed'}
-            />
-            <Button
-                type="submit"
-                variant="contained"
-            >
-                Sign In
-            </Button>
-            {status === "failed" && message &&
-                <Alert sx={{ mt: 1 }} severity="error">{message}</Alert>}
+        <Box sx={{height: '100vh', display: 'flex', alignItems: 'center', justifyContent: 'center'}}>
```

Comment:
I just added a surrounding box and some styles and tabbed everything over. I didn't change the entire file lol

Reply:
that's fine